### PR TITLE
add color coding to cards

### PR DESCRIPTION
**Pull request by On-Fi**

### CODE REVIEW

File: components/Themes.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export default {
+export default { //Fashion: #F9C858 Garden: #858F87 Home: #C4B8AA Bathroom: ##C8D7EE Kitchen: #F4A698
```
Comment (CJAkkuec) [inline components/Themes.js]:
```suggestion
export default {
```
Do you still need this?

File: pages/index.js [hunk 2]
```diff
@@ -71,6 +71,7 @@ export default function HomePage({ projects, favorites, onToggleFavorite, theme
         projectsToDisplay={projectsToDisplay}
         favorites={favorites}
         onToggleFavorite={onToggleFavorite}
+        isWhite={true}
```
Comment (CJAkkuec) [inline pages/index.js:74]: What

**Review by CJAkkuec — APPROVED** [review]
This is approved ... I think? The commented out colors and `isWhite` aside, which I don't understand at all. 😂

File: pages/index.js (hunk 2 above)
Comment (On-Fi) [reply pages/index.js:74]: I try to explain it, 😆
"isWhite" is for the Short-fact-boxes in the homepage, 
the color for the category "fashion" clashes with the color of the Short-fact-boxes,
so we made Short-fact-boxes white but only in the homepage.

**Pull request merged**